packages/esm-patient-forms-app/src/forms/form-view.component.tsx

```diff
@@ -39,15 +39,16 @@ function launchFormEntry(
   currentVisit: Visit | undefined,
   formUuid: string,
   patient: fhir.Patient,
+  htmlFormEntryForms: Array<HtmlFormEntryForm>,
   encounterUuid?: string,
 ) {
   if (currentVisit) {
-    const htmlForm = findHtmlForm(formUuid);
-    if (isEmpty(htmlForm)) {
+    const htmlFormCfg = htmlFormEntryForms.find((form) => form.formUuid === formUuid);
```

Comment:
Wouldn't this variable be better named `htmlForm`?

Reply:
Agreed. Renamed.